### ISSUE TITLE
Guest dashboards show private data like computer and user name for everyone

### ISSUE THREAD

**Issue by ghpqans** [Kitware/CDash#1267]
Even for non logged in users, CDash shows private data like computer and user name. It cannot be removed.
![grafik](https://user-images.githubusercontent.com/56254406/187880649-61b51ce1-2dc0-49bd-9626-b0288c377f47.png)

![grafik](https://user-images.githubusercontent.com/56254406/187880583-089f042f-1d19-4564-8492-2f385f427e37.png)

The reason for that I submit this issue is that I stepped through a CMake tutorial on https://cmake.org/cmake/help/latest/guide/tutorial/index.html up to the section https://cmake.org/cmake/help/latest/guide/tutorial/Adding%20Support%20for%20a%20Testing%20Dashboard.html.
I was quite surprised that in the dashboard I could see my computer and my username and was not able to delete it again.
This was quite astonishing. Nobody told before.

Expected:
Private data should not be shown for everyone. Only for those members of a certain project.

**Comment by talosgl** [comment]
I agree; is there any way to request the removal of these entries once submitted?
